src/main/java/net/mcreator/element/types/Biome.java

```diff
@@ -51,6 +52,7 @@
 	public Color airColor;
 	public Color grassColor;
 	public Color waterColor;
+	public Color waterFogColor;

```

Comment:
Add test case for this in https://github.com/MCreator/MCreator/blob/5fa77237a5f1512baf9d1f42dab8cd92a815a998/src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java#L204